web-console/future-openshift/chargeback/chargeback.md

```diff
@@ -2,20 +2,27 @@
 
 ## Reports
 ![](img/reports-list.png)
```

Comment:
Is name the only sortable field?

Reply:
No, I just haven't updated my sketch file since the new tables went in